pkg/bloomcompactor/metrics.go

```diff
@@ -24,20 +26,23 @@ type Metrics struct {
 	compactionCompleted *prometheus.CounterVec
 	compactionTime      *prometheus.HistogramVec
 
-	tenantsDiscovered    prometheus.Counter
-	tenantsOwned         prometheus.Counter
-	tenantsSkipped       prometheus.Counter
-	tenantsStarted       prometheus.Counter
-	tenantsCompleted     *prometheus.CounterVec
-	tenantsCompletedTime *prometheus.HistogramVec
-	tenantsSeries        prometheus.Histogram
+	tenantsDiscovered   prometheus.Counter
+	tenantsOwned        prometheus.Counter
+	tenantsSkipped      prometheus.Counter
+	tenantsStarted      prometheus.Counter
+	tenantTableRanges   *prometheus.CounterVec
+	seriesPerCompaction prometheus.Histogram
+	bytesPerCompaction  prometheus.Histogram
```

Comment:
I chose to avoid this because per tenant histograms have a way of exploding cardinality and I felt the tenant label isn't likely a necessary one. A regular histogram is a nice middle ground, allowing us to debug in aggregate and there are still log lines we can use to determine per tenant behavior.